packages/onboarding/src/utils/domain-suggester.ts

```diff
@@ -0,0 +1,108 @@
+const validDomains = [
+	'gmail.com',
+	'yahoo.com',
+	'hotmail.com',
+	'aol.com',
+	'outlook.com',
+	'comcast.net',
+	'icloud.com',
+	'msn.com',
+	'hotmail.co.uk',
+	'sbcglobal.net',
+];
```

Comment:
Let's link the source of these domains.

```suggestion
// Taken from: https://www.gmass.co/domains
// These cover ~88% of email addresses.
const validDomains = [
	'gmail.com',
	'yahoo.com',
	'hotmail.com',
	'aol.com',
	'outlook.com',
	'comcast.net',
	'icloud.com',
	'msn.com',
	'hotmail.co.uk',
	'sbcglobal.net',
];
```